notebooks/official/pipelines/multicontender_vs_champion_deployment_method.ipynb

```diff
@@ -757,7 +688,7 @@
         "\n",
         "Next, upload the model's evaluation from the custom training job to the corresponding entry in the Vertex AI Model Registry.\n",
         "\n",
-        "Currently, there is not yet support for this method in the SDK. Instead, you use the lower level GAPIC API interface."
+        "Currently, there is no support for this method yet in the Python SDK. Instead, you use the lower level GAPIC API interface."
```

Comment:
suggested edit:

- Currently, the Python SDK does not support for this method. Instead, use the lower level GAPIC API interface.